test/colony-network.js

```diff
@@ -90,129 +104,126 @@ contract("ColonyNetwork", accounts => {
   describe("when creating new colonies", () => {
     it("should allow users to create new colonies", async () => {
       const token = await Token.new(...TOKEN_ARGS);
-      const { logs } = await colonyNetwork.createColony(token.address);
+      const { logs } = await colonyNetwork.createColony(token.address, 100);
       const { colonyAddress } = logs[0].args;
       const colonyCount = await colonyNetwork.getColonyCount();
       assert.notEqual(colonyAddress, 0x0);
-      assert.equal(colonyCount.toNumber(), 1);
+      expect(colonyCount).to.eq.BN(2);
```

Comment:
I would mildly prefer the change in testing style to be a separate PR (perhaps part of a more comprehensive test suite refactor) but I'll survive.

Reply:
I would normally agree, but I just couldn't help it! Comparing numbers drives me up a wall. I'll try to keep my hands off `bn-chai` for now 👐